guidelines/contributing.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Instead of `guidelines ` folder name, please use `docs` or `.github` so as to follow paths referred by GitHub: https://github.com/metabrainz/design-system/community